packages/kit/src/core/prerender/crawl.js

```diff
@@ -160,6 +160,8 @@ export function crawl(html) {
 								href = value;
 							} else if (name === 'id') {
 								ids.push(value);
+							} else if (name === 'name' && tag === 'A') {
+								ids.push(value);
```

Comment:
nit: the previous suggestion allows us bail out of the `if-else` chain sooner — if `name === 'name'` then we don't want to also test for `name === 'rel'` etc regardless of whether `tag === 'A'`

```suggestion
							} else if (name === 'name') {
								if (tag === 'A') ids.push(value);
```

Reply:
since this was merged i'm just gonna push this change direct to master, don't yell at me